include/gz/sim/sim.hh.in

```diff
@@ -0,0 +1,3 @@
+// Automatically generated
+#include <gz/<project-name>/config.hh>
```

Comment:
This doesn't look right, I don't think this file is being used?

Reply:
I've opened https://github.com/gazebosim/gz-sim/pull/1490 to delete this file in ign-gazebo3. I think we can just delete it here as well so we don't have to wait for the forward-port

Reply:
Cross-post from my comment in #1490 

The ignition/gazebo/gazebo.hh.in was renamed to gz/sim/sim.hh.in here.

I suppose I should create a gazebo.hh.in in gz/sim/ though since CMake is currently using the gazebo project name which is causing gazebo.hh to be generated.

Reply:
`gz/sim/gazebo.hh.in` has been created in https://github.com/gazebosim/gz-sim/pull/1477/commits/146d92fbcb9b339f7d9d96e1c5630a59449bd3ca

It'll be used as long as the project name remains `ignition-gazebo7`. When we migrate the project name we can delete it.

Reply:
I should have been more specific in my comment that I think the `<project-name>` string is the problem - that doesn't seem to be using the correct syntax for substitution. I also don't think this template is actually being configured anywhere.

Reply:
---

After some conversation on Slack, I've deleted both `sim.hh.in` and `gazebo.hh.in` on the branch for both `ignition` and `gz`!